CONTRIBUTING.md

```diff
@@ -78,3 +78,5 @@ $ ./format.sh
 ```
 
 If `format.sh` has any problems with your code, go and fix the issues. If you can't fix a warning without making the function not match anymore, append `// NOLINT` to the offending line.
+
+Then, please [create a pull request](https://github.com/pmret/nananian/pulls)!
```

Comment:
this seems to have your username hardcoded?

Reply:
oooooooooooooooooops

Reply:
good spot!